### DISCUSSION TITLE
Diablo 3 Reaper of Souls Ultimate Evil Edition CUSA00242 v01.42

### DISCUSSION THREAD

**Discussion by st0nec0ldrem0rse** [GoldHEN/GoldHEN_Cheat_Repository#180] [Cheat requests]
I kindly request for a cheat to be updated and added on GoldHen Cheat for the above game version 
many Thanks and more power Guys!!

**Comment by st0nec0ldrem0rse** [comment]
any updates on this request please?

**Reply by kmeps4** [reply]
you will have to request for that cheat on PS4 Cheater or Team Reaper Discord since there's no public one made by a cheat dev.
